assets/admin_themes/Sea_Green/navbar/navbar.scss

```
@@ -78,130 +108,125 @@
-    &:focus-within,
-    &:focus,
-    &:active
-    {
-      color: $g-800;
-      outline: none;
-      border-color: $grape;
-      background-color: $white;
+        // Style for dropdown-item inside
+        .dropdown-item {
+            @include state-border(1px, transparent);
+            // Hover state - grey background, no border
```

Stylelint (17.11.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)